### PR TITLE
Get rid of rubbish logs from serialization_test

### PR DESCRIPTION
**Pull request by wojtek-t**

### CODE REVIEW

File: pkg/api/serialization_test.go [hunk 1]
```diff
@@ -122,8 +121,8 @@ func roundTripSame(t *testing.T, item runtime.Object, except ...string) {
 
 // For debugging problems
 func TestSpecificKind(t *testing.T) {
-	api.Scheme.Log(t)
-	defer api.Scheme.Log(nil)
+	// api.Scheme.Log(t)
```
Comment (gmarek) [inline pkg/api/serialization_test.go]: Add a comment explaining why you leave this code commented instead of removing it. Same below.

Comment (wojtek-t) [inline pkg/api/serialization_test.go]: I did it the same way it was already commented in one of the tests below: TestRoundTripTypes

Comment (gmarek) [inline pkg/api/serialization_test.go]: Oh, OK.

**Comment by wojtek-t** [comment]
Trivial fix. Reapplying label.

**Comment by gmarek** [comment]
`FAIL	k8s.io/kubernetes/pkg/storage	8.350s`
@k8s-bot unit test this

**Comment by wojtek-t** [comment]
@k8s-bot unit test this please

**Comment by wojtek-t** [comment]
@k8s-bot unit test this please

**Pull request merged**